### DISCUSSION TITLE
[小工具][可选] 修复 1.6.1-1.7.2 Forge 无法使用 Java 8 启动的特性

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#4207] [讨论]
### 虽然现在可能没啥人有这需求，不过还是发一下比较好
java.com 已经没有 Java 7 的下载链接了，Oracle JDK 归档的版本好像也不太能用(卡在运行)？
实际上原版是支持 Java 8 的，但是因为 Forge 适用于 1.6.1-1.7.2 的版本在代码和 Java 8 有严重的兼容性问题，所以安装了 Forge 版本的 Minecraft 1.6.1-1.7.2 只能使用 Java 7
LexManos 提供的修复办法就是安装这个补丁
### 此补丁允许游戏运行在较高的 Java 版本（8u412）

[MC 百科链接](https://www.mcmod.cn/class/8621.html)
[Forge Maven 下载地址](https://maven.minecraftforge.net/net/minecraftforge/lex/legacyjavafixer/1.0/legacyjavafixer-1.0.jar)

**Comment by Pigeon0v0** [comment]
虽然说不再对 1.9 以下版本特别制作功能，但这个我觉得可以加一下...

**Comment by 3gf8jv4dv** [comment]
From my personal experience, Forge 1.7.10-pre4 also has the same problem (Although almost no one uses this version now).

> 所以安装了 Forge 版本的 Minecraft 1.6.1-1.7.2 只能使用 Java 7

There are also versions of Java 8 that allow the game to run, but they are very old (less than 8u20). Binaries can be found at the Internet Archive.

**Reply by wuliaodexiaoluo** [reply]
In fact, running older versions of Java to play games can lead to a number of security issues
This patch allows versions 1.6.1-1.7.2 to run on a higher Java version (8u412), which will be more secure :)

**Reply by 3gf8jv4dv** [reply]
> In fact, running older versions of Java to play games can lead to a number of security issues This patch allows versions 1.6.1-1.7.2 to run on a higher Java version (8u412), which will be more secure :)

Agree with you. I am just mentioning it. :)
